bundles/core/pom.xml

```diff
@@ -529,6 +529,11 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.adobe.granite.bundles</groupId>
+            <artifactId>json</artifactId>
```

Comment:
This is really deprecated, rather use JSON-P Impl with implementation Jackson.

Reply:
@kwin , unfortunately we cannot get rid completely of `org.json` as it's part of a public API: https://github.com/adobe/aem-core-wcm-components/blob/main/bundles/core/src/main/java/com/adobe/cq/wcm/core/components/services/form/FormHandler.java#L39

Reply:
How about deprecating this method or interface in favor of anew API?